src/app/tab-selector/tab-selector.component.html

```diff
@@ -6,18 +6,35 @@
   <div
     *ngFor="let tab of tabs"
     (click)="_tabClicked(tab)"
-    class="d-flex flex-column align-items-center h-100 pl-15px pr-15px"
+    class="d-flex flex-column align-items-center h-100 pl-15px pr-15px tab-selector__tab"
+    [ngClass]="{ highlighted: highlightTab && tab === activeTab }"
   >
     <div class="d-flex h-100 align-items-center">
       <div
         class="d-flex h-100 align-items-center fs-15px"
         *ngIf="tab !== activeTab || !buttonSelector"
         [ngClass]="{ 'active-tab': tab === activeTab, 'fc-secondary': tab !== activeTab }"
       >
-        {{ tab | transloco }}
+        <ng-container
```

Comment:
What is this change doing? This component is used pretty broadly across the app, we should be sure we're not breaking any other functionality before modifying it.

Reply:
This actually extends the functionality of `tab-selector` without modifying the existing logic.

With this `ng-container` you are able to pass you own custom code of a tab item (in my case it's an emoji with the counter), that was not possible to do before, because `tab-selector` was only accepting strings.

So now I use it like this:
```
    <tab-selector
      [tabs]="reactionTabs"
      [activeTab]="activeReactionTab"
      [highlightTab]="true"
      (tabClick)="selectTab($event)"
    >
      <ng-template let-tabItem #tabItem>
        <div>
          <reactions-image [reaction]="tabItem" [size]="20"></reactions-image>
          <b class="ml-2">{{ postReactionCounts.Counts[tabItem] }}</b>
        </div>
      </ng-template>
    </tab-selector>
```